src/main/java/seedu/address/logic/commands/MailIndexCommand.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.MailUtil.launchMail;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Opens the system default mail app with the person
+ * identified by the index specified by the user.
+ */
+public class MailIndexCommand extends Command {
+
+    public static final String COMMAND_WORD = "mail-index";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Opens the mailbox to email the person identified by the "
+            + "index number used in the displayed person list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Opening System Default Mail Application : %1$s";
+
+    private final Index targetIndex;
+
+    /**
+     * Constructor for the {@code MailIndexCommand} class.
+     * @param targetIndex The index of the person in the contact list.
+     */
+    public MailIndexCommand(Index targetIndex) {
+        this.targetIndex = targetIndex;
+    }
+
+    /**
+     * Executes the mail-s command.
```

Comment:
Might want to update this to mail-index instead of mail-s!

Same thing for MailXCommand